### DISCUSSION TITLE
Display build message like a CI

### DISCUSSION THREAD

**Discussion by ice1000** [community/community#21876] [Other Feature Feedback, Questions, & Ideas]
<p>Can GitHub Pages display build messages like a CI (Travis, AV, Circle, etc.) when building with Jekyll? Faced some weird problem in the last few days.</p>

**Comment by darrowby385** [comment, marked as answer]
<p>Hi <a class="mention" href="/ice1000">@ice1000</a>!</p>
<p>It isn’t currently possible to configure CI-style messages for the GitHub Pages build process, though it’s definitely a neat idea!</p>
<p>If you’re having trouble with a GitHub Pages build, the best way to dig a bit deeper and debug it is to configure a GitHub Pages-optimised Jekyll instance on your local machine. Once you’ve done that, you can run test builds locally and view the full errors/warnings generated by Jekyll.</p>
<p>There’s documentation here that explains how to get this set up:</p>
<p><a href="https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/" rel="nofollow noopener">https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/</a></p>
<p>I hope that helps!</p>

**Comment by ice1000** [comment]
<p>Thanks!</p>
